compiler/src/dmd/inlinecost.d

```diff
@@ -73,22 +73,6 @@ int inlineCostExpression(Expression e)
 }
 
 
-/*********************************
- * Determine cost of inlining function
- * Params:
- *      fd = function to determine cost of
- *      hasthis = if the function call has explicit 'this' expression
- *      hdrscan = if generating a header file
- * Returns:
- *      cost of inlining fd
- */
-int inlineCostFunction(FuncDeclaration fd, bool hasthis, bool hdrscan)
```

Comment:
Why is the whole of this file not removed too? as far as I can tell it is only used by `inline.d` which is completely removed by this PR.

Reply:
I already wrote this in the OP

> Parts of the front-end inliner is still kept around for inlining default arguments (inlineCopy).

See expressionsem.d for the reference.